src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,167 @@
+                + " but it wasn't");
+    }
+
+    @Test
+    void register_nullPassword_notOk() {
+        user.setPassword(null);
+        assertThrows(RegistrationFailedException.class, () -> {
+            registrationService.register(user);
+        }, " expected " + RegistrationFailedException.class.getName()
+                + " to be thrown for null password "
+                + user.getPassword()
+                + " but it wasn't");
+        user.setPassword(EMPTY_ENTRY);
+        assertThrows(RegistrationFailedException.class, () -> {
+            registrationService.register(user);
+        }, " expected " + RegistrationFailedException.class.getName()
+                + " to be thrown for empty entry password "
+                + user.getPassword()
+                + " but it wasn't");
+    }
+
+    @Test
+    void register_lessLengthPassword_notOk() {
+        user.setPassword("qwer");
+        assertThrows(RegistrationFailedException.class, () -> {
+            registrationService.register(user);
+        }, " expected " + RegistrationFailedException.class.getName()
+                + " to be thrown for less length password "
+                + user.getPassword()
+                + " but it wasn't");
+    }
+
+    @Test
+    void register_overLengthPassword_notOk() {
+        user.setPassword("qwerty123456789012345678901234567890");
+        assertThrows(RegistrationFailedException.class, () -> {
+            registrationService.register(user);
+        }, " expected " + RegistrationFailedException.class.getName()
+                + " to be thrown for over password length "
+                + user.getPassword()
+                + " but it wasn't");
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        user.setAge(null);
+        assertThrows(RegistrationFailedException.class, () -> {
+            registrationService.register(user);
+        }, " expected " + RegistrationFailedException.class.getName()
+                + " to be thrown for null age "
+                + user.getAge()
+                + " but it wasn't");
+    }
+
+    @Test
+    void register_lessAge_notOk() {
+        user.setAge(17);
+        assertThrows(RegistrationFailedException.class, () -> {
+            registrationService.register(user);
+        }, " expected " + RegistrationFailedException.class.getName()
+                + " to be thrown for less age "
+                + user.getAge()
+                + " but it wasn't");
+    }
+
+    @Test
+    void register_overAge_notOk() {
+        user.setAge(101);
+        assertThrows(RegistrationFailedException.class, () -> {
+            registrationService.register(user);
+        }, " expected " + RegistrationFailedException.class.getName()
+                + " to be thrown for over age "
+                + user.getAge()
+                + " but it wasn't");
+    }
+
+    @Test
+    void register_successful_ok() {
+        registrationService.register(user);
+        Storage.people.add(user);
```

Comment:
this is useless line, cause `registrationService.register(user);` have already added new user